system/ChunkAllocator.cpp

```diff
@@ -96,7 +96,7 @@ static void _aligned_allocator_append_chunk(struct aligned_allocator *aa,
     shared_pool_total_allocated += std::max( new_chunk->chunk_size, (decltype(chunk_size)) CACHE_LINE_SIZE );
     
     // for timing only
-    memset(new_chunk->chunk, new_chunk->chunk_size, 0);
+    memset(new_chunk->chunk, 0, new_chunk->chunk_size);
```

Comment:
This was something I merged from Mark's SWIG code, and was "as designed" before. As the comment implies, it was intended to call memset and do no work. I'm not sure why, at this point. I didn't touch it since we were working towards a deadline. 

I would actually suggest removing the memset, since the buffer doesn't need to be zeroed and thus this will just add overhead. If you want to keep it, lose the comment.